dashboard/v1.1/INSTALL.md

```diff
@@ -1,5 +1,24 @@
 ### Installation Instructions
 
-Please follow Makefile in the root directory of the project to install the UI.
+Please follow the following steps in order to set up your
+development environment.
+
+1. Install NodeJS and yarn.
+2. Clone the repository or its fork via 
+   ```
+   git clone https://github.com/zairza-cetb/bench-routes
+   ```
+3. Navigate into the cloned repo 
+   ```
+   cd bench-routes
+   ```
```

Comment:
Hi @maheshschand, I dont think the instructions here are valid. Please refer to the `Makefile` in the root directory to clear your doubts. Please revert the changes in this file. Thanks.

Reply:
OK.